modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/system/CommerceOrderSystemObjectDefinitionManager.java

```diff
@@ -129,6 +129,12 @@ public List<ObjectField> getObjectFields() {
 			createObjectField(
 				"Integer", "Integer", "order-status", "orderStatus", true,
 				true),
+			createObjectField(
+				"Text", "String", "orderTypeExternalReferenceCode",
+				"orderTypeExternalReferenceCode", true, true),
+			createObjectField(
+				"LongInteger", "Long", "order-type-id", "orderTypeId", true,
+				true),
```

Comment:
@joanagrego to add these new fields correctly, you need to increase the version of this system object to 3, like we did [here](https://github.com/liferay/liferay-portal/commit/1960d0172f002a6a56a5125645a2146e00a757a2#diff-9e383eb4fdc0d8950c2709dde6043b95c64cd2e68a395d92fc1c4822fec4af07R130)

Reply:
Done.